src/components/ContentPreview/ContentPreview.js

```diff
@@ -73,7 +73,8 @@ type Props = {
     onMetric?: Function,
     requestInterceptor?: Function,
     responseInterceptor?: Function,
-    previewInstance?: any
+    previewInstance?: any,
+    getUserProfileUrl?: Function
```

Comment:
Shouldn't this be in `contentSidebarProps`?

Reply:
Good point. Testing that out now

Reply:
@jeremypress fixed it